workflows/src/test/kotlin/com/r3/developers/JsonSerializers.kt

```diff
@@ -0,0 +1,21 @@
+package com.r3.developers
+
+import com.fasterxml.jackson.core.JsonGenerator
+import com.fasterxml.jackson.core.JsonParser
+import com.fasterxml.jackson.databind.DeserializationContext
+import com.fasterxml.jackson.databind.JsonDeserializer
+import com.fasterxml.jackson.databind.JsonSerializer
+import com.fasterxml.jackson.databind.SerializerProvider
+import net.corda.v5.base.types.MemberX500Name
+
+object MemberX500NameSerializer : JsonSerializer<MemberX500Name>() {
+    override fun serialize(value: MemberX500Name, generator: JsonGenerator, provider: SerializerProvider?) {
+        generator.writeString(value.toString())
```

Comment:
where is the output of the serialization ?

Reply:
We're overriding a function from Jackson > JsonSerializer which handles the generator and the value it produces internally

Reply:
thanks for clarifying.